src/PresentationalComponents/Email/Email.js

```diff
@@ -89,15 +116,27 @@ const Email = () => {
                                 <div>Email subscriptions</div>
                                 <div className="pref-email_subheader">Select the cloud.redhat.com emails you want to receive.</div>
                             </CardHeader>
-                            <CardBody>
+                            <CardBody className="pref-email_form">
                                 <FormRender
+                                    keepDirtyOnReinitialize
                                     formFieldsMapper={ {
                                         ...formFieldsMapper,
                                         [DESCRIPTIVE_CHECKBOX]: DescriptiveCheckbox,
-                                        [LOADER]: Loader
+                                        [LOADER]: Loader,
+                                        [DATA_LIST]: DataListLayout
                                     } }
                                     layoutMapper={ layoutMapper }
-                                    schema={ schema }
+                                    schema={ {
+                                        fields: [{
+                                            name: 'email-preferences',
+                                            component: DATA_LIST,
+                                            sections: Object.entries(email).map(([ key, schema ]) => ({
+                                                label: schema?.title,
+                                                name: key,
+                                                fields: schema.fields || getSchema(store?.[key])
```

Comment:
`schema?.fields` or delete the `?` above from `schema?.title`. Depends if the `schema` can be empty.